modules/javafx.graphics/src/main/java/javafx/scene/SpotLight.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Normally we do not change the start year, however it's OK in this case, since it is correcting an earlier mistake.